.github/workflows/main.yaml

```diff
@@ -0,0 +1,48 @@
+name: main
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['*']
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '17'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn clean package
+  deploy-scipooldev:
+    if: github.ref == 'refs/heads/develop'
+    needs: [test]
+    uses: "./.github/workflows/aws-deploy.yaml"
+    with:
+      role-to-assume: "arn:aws:iam::465877038949:role/sagebase-github-oidc-scipool-dev-synapse-login-aws-infra"
+      ebcli-command: "deploy --profile develop --verbose --region ${AWS_REGION} --staged --timeout 30 synapse-login-scipooldev"
```

Comment:
Do we still need to set the profile here? In the travis job, profiles were added to `.aws/config` to choose the correct role, but in this github action we're setting the role explicitly. Also, I think `AWS_REGION` is unset here.

Reply:
nice catch. fixed, thanks.